server/vcs/provider/github/reviewer_fetcher.go

```diff
@@ -0,0 +1,53 @@
+package github
+
+import (
+	"context"
+	"fmt"
+	gh "github.com/google/go-github/v45/github"
+	"github.com/palantir/go-githubapp/githubapp"
+	"github.com/pkg/errors"
+	"github.com/runatlantis/atlantis/server/events/models"
+	"net/http"
+)
+
+const ApprovalState = "APPROVED"
+
+type PRReviewerFetcher struct {
+	ClientCreator githubapp.ClientCreator
+}
+
+func (r *PRReviewerFetcher) ListApprovalReviewers(ctx context.Context, installationToken int64, repo models.Repo, prNum int) ([]string, error) {
+	client, err := r.ClientCreator.NewInstallationClient(installationToken)
+	if err != nil {
+		return nil, errors.Wrap(err, "creating installation client")
+	}
+
+	var approvalReviewers []string
+	nextPage := 0
+	for {
+		listOptions := gh.ListOptions{
+			PerPage: 300,
```

Comment:
```suggestion
			PerPage: 100,
```

The GitHub API docs say it only allows a max of 100 for this value.